.gitignore

```diff
@@ -0,0 +1,2 @@
+main
+rudolph
```

Comment:
@kalmanb, I get the main is excluding the binary, what file is "rudolph" ignoring?

Reply:
When you check it out it should be in <GOPATH>/src/github.com/dhruv11/rudolph
if you go `go build` it will create `rudolph`

Typically you'd have a `Makefile` to do builds, scripts etc and give the binary a name. eg https://sohlich.github.io/post/go_makefile/